op-service/txmgr/txmgr.go

```diff
@@ -320,6 +331,60 @@ func (m *SimpleTxManager) craftTx(ctx context.Context, candidate TxCandidate) (*
 	return m.signWithNextNonce(ctx, txMessage) // signer sets the nonce field of the tx
 }
 
+func (m *SimpleTxManager) GetMinBaseFee() *big.Int {
+	return m.cfg.MinBaseFee.Load()
+}
+
+func (m *SimpleTxManager) SetMinBaseFee(val *big.Int) {
+	m.cfg.MinBaseFee.Store(val)
+	m.l.Info("txmgr config val changed: SetMinBaseFee", "newVal", val.String())
```

Comment:
You don't need to access `String()` of a `big.Int`, it should be logged in its string representation automatically. Applies to other logs as well.